sass/README.md

```diff
@@ -0,0 +1,41 @@
+SASS Style Guide
+=====
+### Formatting
+* Use the SASS syntax.
```

Comment:
I much rather prefer the SASS syntax for almost all cases. I can think of maps, that don't look as good. But I think that's it.

In Rails projects, is very easy to have both, and use SCSS in a variables file, for instance, and SASS for the rest. I think that's what we did on easy.money, and I don't know of any reasons to complain about that approach.

Have you found any other use case in which SCSS might actually look nicer or be easier to read than SASS?